### PR TITLE
Fix check if object is null

### PR DESCRIPTION
**Pull request by lamrani002**
# FIX|Fix phan warning

### CODE REVIEW

**Comment by eldy** [comment]
What was the warning code ?

**Comment by lamrani002** [comment]
Property CommonDocGenerator::$extrafieldsCache (ExtraFields) in empty() is not falsy.

File: htdocs/core/class/commondocgenerator.class.php [hunk 1]
```diff
@@ -1368,7 +1368,7 @@ public function getExtrafieldContent($object, $extrafieldKey, $outputlangs = nul
 
 
 		// Load extra fields if they haven't been loaded already.
-		if (empty($this->extrafieldsCache)) {
```
Comment (eldy) [inline htdocs/core/class/commondocgenerator.class.php:1371]: Try with if (!is_object...

**Review by eldy — CHANGES_REQUESTED** [review]

**Comment by eldy** [comment]
Pr has meege conflict, surely because of fix already meeged differently. So i close pr.

**Pull request closed without merging**